File: Clients/BlogBlazor/Shared/Blazor.Contracts/V1/Requests/TagsRequests/CreateTagRequest.cs
namespace Blazor.Contracts.V1.Requests.TagsRequests;

using Interfaces;

/// <summary>
/// Create tag request.
/// </summary>
public class CreateTagRequest : IRequest
{
    /// <summary>
    /// Gets or sets the title.
    /// </summary>
    /// <value>
    /// The title.
    /// </value>
    public string Title { get; set; }
}

File: Shared/Blog.FSharp.Contracts/V1/Requests/TagsRequests/CreateTagRequest.fs
namespace Blog.FSharp.Contracts.V1.Requests.TagsRequests

open Blog.FSharp.Contracts.V1.Requests.Interfaces

/// <summary>
/// Create tag request.
/// </summary>
type CreateTagRequest() =
    interface IRequest

    /// <summary>
    /// Gets or sets the title.
    /// </summary>
    member val Title : string = null with get, set